### ISSUE TITLE
Unable to set PublishingDate for flac files

### ISSUE THREAD

**Issue by nlogozzo** [Zeugma440/atldotnet#235]
Hi,
We have this flac file: https://we.tl/t-5ZnMpOASFi
*Sorry for the wetransfer link but the file is bigger than 20 MB so GitHub doesn't like it. If it expires before the time you download it, let me know I will send a new one.*

We simply load it and set the PublishingDate:
```csharp
var track = new Track(Path);
track.PublishingDate = new DateTime(2023, 10, 18);
track.Save();
```

upon reloading and reading the file, the publish date was not saved:
```csharp
var track = new Track(Path);
Console.WriteLine(track.PublishingDate?.ToShortDateString() ?? ""); //Nothing is printed 
```

From our testing, this issue only affects `flac` files and not `mp3`.

**Comment by Zeugma440** [comment]
Explanation's simple : Vorbis didn't have any mapping for that field until now.

Now it's done 😉 

I took advantage of your request to map WMA as well, which was [a little more complicated, spec-wise](https://www.mediamonkey.com/forum/viewtopic.php?f=7&t=42797).

**Comment by nlogozzo** [comment]
Thanks!!

**Comment by nlogozzo** [comment]
If we could get a release, that would be great :)

**Comment by Zeugma440** [comment]
Available on today's v5.11 👍
